src/plugins/mer/mermanagementwebview.cpp

```diff
@@ -347,17 +348,18 @@ void MerManagementWebView::handleLoadFinished(bool success)
                     "  %3"
                     "  </div>"
                     "  <div style='padding-left: 24px; padding-right: 24px; font-size: small;'>"
-                    "    <p>A Sailfish OS build engine can be controlled with the "
+                    "    <p>A %5 build engine can be controlled with the "
                     "    <img src='qrc%4' style='vertical-align: middle;'/>"
                     "    button &ndash; available on the lower left side <em>when "
-                    "    a&nbsp;Sailfish&nbsp;OS project is open</em>.</p>"
+                    "    a&nbsp;%5&nbsp; project is open</em>.</p>"
                     "  </div>"
                     "</body>"
                     "</html>")
                 .arg(qApp->font().family())
                 .arg(qApp->font().lastResortFamily())
                 .arg(vmStatus)
                 .arg(Icons::MER_SDK_RUN.imageFileName())
+                .arg(Mer::Constants::MER_VENDOR_OS_NAME)
```

Comment:
Looks like some forgotten occurrence. BTW control centre embedding will be removed soon https://github.com/sailfishos/sailfish-qtcreator/pull/228

Reply:
Undo changes with control centre